ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CheckDetailsPage.tsx

```diff
@@ -31,42 +32,45 @@ function sortCheckStats(a: ComplianceCheckStatusCount, b: ComplianceCheckStatusC
 function CheckDetails() {
     const { checkName, profileName } = useParams();
     const [currentDatetime, setCurrentDatetime] = useState(new Date());
+    const pagination = useURLPagination(10);
```

Comment:
What are pro and con of a constant for page size like this:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/constants.ts#L1

Maybe factor out to the level of a folder like Coverage or Schedules to balance consistency with flexibility?

Reply:
I think that's a good idea. I was planning on making that constants file for the sortable/searchable fields as well. Seems reasonable to include all of that in the same file.